runner.rb

```diff
@@ -0,0 +1,16 @@
+# runner.rb
```

Comment:
Copied and pasted all the code from iteration 1 page

Reply:
This looks great! Good work on this runner file!